src/cts/src/HTreeBuilder.cpp

```diff
@@ -123,6 +123,14 @@ void HTreeBuilder::preSinkClustering(
                                   options_->getSinkBuffer(),
                                   legalCenter.getX() * wireSegmentUnit_,
                                   legalCenter.getY() * wireSegmentUnit_);
+      if (logger_->debugCheck(CTS, "legalizer", 2) && (center != legalCenter)) {
+        // clang-format off
+	debugPrint(logger_, CTS, "legalizer", 2,
+		   "preSinkClustering legalizeOneBuffer {}: {} => {}",
+		   baseName + std::to_string(clusterCount),
+		   center, legalCenter);
+        // clang-format on
```

Comment:
Why the `clang-format` comments? Here and in the other debugPrints

Reply:
clang wants to expand debugPrint line into like 10+ lines.  I want to make debug statements compact so this doesn't impact  readability of the rest of the codes.